test/model_test.py

```diff
@@ -20,20 +20,24 @@ def make_spectrogram(seconds):
 class ModelTest(unittest.TestCase):
     def test_model_make(self):
 
-        model.make()
+        model.make(2)
 
     def test_model_test(self):
         mock_input = np.asarray([make_spectrogram(5), make_spectrogram(5)])
         mock_output = np.array([[0.0, 1.0], [1.0, 0.0]])
 
-        the_model = model.make()
+        the_model = model.make(len(mock_input))
 
         model.test(the_model, mock_input, mock_output)
 
     def train_model_test(self):
         mock_input = np.asarray([make_spectrogram(5), make_spectrogram(5)])
         mock_output = np.array([[0.0, 1.0], [1.0, 0.0]])
 
-        the_model = model.make()
+        the_model = model.make(len(mock_input))
 
         model.train(the_model, mock_input, mock_output, batch_size=1, epochs=5)
+
+
+rand = ModelTest()
+rand.test_model_test()
```

Comment:
This is all great :) looks like it will work!
but I don't think you need these lines here! the unit test runner runs all the tests anyway